### PR TITLE
Add Ingress example with Minikube

### PR DESCRIPTION
**Pull request by nastra**

### CODE REVIEW

**Review by XN137 — APPROVED** [review]

File: helm/nessie/README.md [hunk 1]
```diff
@@ -83,6 +83,35 @@ aws_secret_access_key=YOURSECRETKEYDATA
...
 * Install Nessie Helm chart: `helm install nessie -n nessie-ns helm/nessie`
 
+### Ingress with Minikube
+
+This is broadly following the example from https://kubernetes.io/docs/tasks/access-application-cluster/ingress-minikube/
+
+* Start Minikube cluster: `minikube start`
+* Enable NGINX Ingress controller: `minikube addons enable ingress`
+* Verify Ingress controller is running: `kubectl get pods -n ingress-nginx`
+* Configure Nessie with the following ingress settings
```
Comment (snazy) [inline helm/nessie/README.md:93]: Where / how would a user do this?

File: helm/nessie/README.md [hunk 2]
```diff
@@ -83,6 +83,35 @@ aws_secret_access_key=YOURSECRETKEYDATA
...
+  ```
+* Create K8s Namespace: `kubectl create namespace nessie-ns`
+* Install Nessie Helm chart: `helm install nessie -n nessie-ns helm/nessie`
+* Verify that the IP address is set:
+  ```
+  $ kubectl get ingress
+  NAME     CLASS   HOSTS   ADDRESS        PORTS   AGE
+  nessie   nginx   *       192.168.49.2   80      4m35s
+  ```
+* Add `192.168.49.2 chart-example.local` to `/etc/hosts`
```
Comment (snazy) [inline helm/nessie/README.md:112]: Is it the recommended practice to update `/etc/hosts`?

Comment (nastra) [reply helm/nessie/README.md:112]: this is just for a local example. In prod you'd also have routing properly configured

File: helm/nessie/README.md (hunk 1 above)
Comment (nastra) [reply helm/nessie/README.md:93]: in the `values.yaml` of the helm chart

Comment (nastra) [reply helm/nessie/README.md:93]: I'll make this more clear in the next PR

File: helm/nessie/README.md (hunk 2 above)
Comment (snazy) [inline helm/nessie/README.md:112]: Also: I suspect the IP address isn't fixed. Maybe add a note to use the IP from the output above (for dummies).

Comment (snazy) [inline helm/nessie/README.md:112]: Replace with 
```
echo "192.168.49.2 chart-example.local" | sudo tee /etc/hosts
```

**Pull request merged**